### ISSUE TITLE
Demo page broken

### ISSUE THREAD

**Issue by neothemachine** [dwilhelm89/LeafletSlider#32]
Tested with latest Firefox. (Hint: https/http mixing)

**Comment by dwilhelm89** [comment]
I can not reproduce any issues. Tested with latest Chrome, Firefox and Internet Explorer. Can you describe the problem in more detail?

**Comment by neothemachine** [comment]
Sorry, it actually only happens when I access it via https: 
https://dwilhelm89.github.io/LeafletSlider/ I just realized that the 
HTTPS Everywhere browser plugin is doing the automatic switch from http 
to https. It seems that the leafletjs cdn has an invalid SSL 
certificate, otherwise it would be as simple as leaving off the protocol 
when including leaflet in your html. I also reported this same issue on 
another plugin (https://github.com/Leaflet/Leaflet.fullscreen/issues/38) 
so maybe it's best to keep any discussion over there.
